### PR TITLE
PY-25689 image Sphinx directive doesn't work in docstring documentation

### PR DESCRIPTION
**Pull request by Eitol**
Fix the problem: the prefix "file: //" was not added to local image references.

image

ref: https://youtrack.jetbrains.com/issue/PY-25689

contributor license agreement -> ready

### CODE REVIEW

**Comment by Eitol** [comment]
hi @ignatov Is there any way to move this forward?

**Comment by ignatov** [comment]
@denofevil and @JB-Dmitry, please look on this.

File: platform/lang-impl/src/com/intellij/codeInsight/documentation/DocumentationComponent.java [hunk 1]
```diff
@@ -776,14 +779,73 @@ private void setDataInternal(@Nullable SmartPsiElementPointer<PsiElement> elemen
...
+   *              then it returns the value passed by parameter
+   */
+  private String fixImageReferences(String text) {
+    if (text == null || text.isEmpty() || !BasicHTML.isHTMLString(text)) {
+      return text;
+    }
+    text = text.replace("<br>", "<br/>");
+    String basePath = this.myManager.myProject.getBasePath();
+    try {
+      org.jdom.Element dom = JDOMUtil.load(text);
```
Comment (denofevil) [inline platform/lang-impl/src/com/intellij/codeInsight/documentation/DocumentationComponent.java]: Sorry, this approach won't work: `text`  is an HTML document with all funny things like unclosed tags and unquoted attribute values, which makes it unreadable by XML parsers.
Also, I don't think that it's a proper place to fix Python specific problem, maybe `com.jetbrains.python.documentation.PythonDocumentationProvider` would be a better place.

Comment (denofevil) [reply platform/lang-impl/src/com/intellij/codeInsight/documentation/DocumentationComponent.java]: @vlasovskikh

**Pull request closed without merging**